src/details/ArborX_DetailsTreeTraversal.hpp

```diff
@@ -192,7 +192,7 @@ struct TreeTraversal<BVH, Predicates, Callback, NearestPredicateTag>
     else if (bvh_.size() == 1)
     {
       Kokkos::parallel_for(
-          "ArborX::TreeTraversal::nearest_queries_degenerated_one_leaf_tree",
+          "ArborX::TreeTraversal::query::nearest_degenerated_one_leaf_tree",
```

Comment:
I think it would be fine to leave this one alone. If you do decide to change it, you would also need to change other places in the file, for example, spatial queries.

But, I do think this file needs a more thorough cleanup. @dalg24 I did not notice at the time, but `TreeTraversal` file has a mix of `TreeTraversal` and `BVH` labels. I want to confirm that it was intentional, and suggest changing all to `TreeTraversal` instead of `BVH`. But it does not have to be done now.

Reply:
Oversight on my part.  Sounded like a good idea at the time.

Reply:
Resolved in #398.